database/mysql/build-baseline.sh

```
@@ -0,0 +1,119 @@
+# Normalize a mysqldump stream so diffs are stable across runs and across the legacy vs Flyway path.
+normalize() {
+  sed -E \
+    -e 's/ AUTO_INCREMENT=[0-9]+//g' \
+    -e '/^-- Dump completed/d' \
+    -e '/^-- Server version/d' \
+    -e '/^\/\*![0-9]+ SET /d'
+}
+
+build_and_dump() {
+  province="$1"   # on | bc
+  icd="$2"        # 9 | 10
```

Shellcheck (0.11.0)

[warning] icd appears unused. Verify use (or export if used externally).

(SC2034)